clients/src/main/java/org/apache/kafka/common/security/oauthbearer/internals/secured/VerificationKeyResolverFactory.java

```diff
@@ -58,6 +58,7 @@ public static CloseableVerificationKeyResolver create(Map<String, ?> configs,
         String saslMechanism,
         Map<String, Object> jaasConfig) {
         ConfigurationUtils cu = new ConfigurationUtils(configs, saslMechanism);
+        cu.throwIfURLIsNotAllowed(SASL_OAUTHBEARER_JWKS_ENDPOINT_URL);
```

Comment:
I wonder if we should call `throwIfURLIsNotAllowed()` from `validateUrl()`. Otherwise all caller have to have this check and we could miss it if we add a new caller in the future.

Reply:
I just thought strictly speaking, `throwIfURLIsNotAllowed` is not part of url validation. But I don't have strong opinion on it. Updated.